bucket/docker-nightly.json

```diff
@@ -31,17 +31,14 @@
         "For help, see https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/configure-docker-daemon"
     ],
     "suggest": {
-        "Linux container support on Windows": "lcow",
-        "Docker Compose": "docker-compose",
-        "Docker Machine": "docker-machine",
-        "Kubernetes Standalone": "minikube",
-        "Kubernetes Client": "kubectl",
-        "compose.yml to kubernetes.yml": "kompose"
+        "sudo": "gsudo",
+        "Docker Compose": "main/docker-compose",
+        "Docker Machine": "main/docker-machine",
+        "Kubernetes Client": "main/kubectl",
+        "Kubernetes Standalone": "main/minikube",
+        "Linux container support on Windows": "unknown/lcow",
```

Comment:
This manifest appears to have been lost. https://github.com/ScoopInstaller/Main/pull/2972

Reply:
I remember putting it somewhere in the `deprecated` folder...

Reply:
After being removed from main bucket, it was not created anywhere.

https://github.com/search?q=org%3AScoopInstaller+lcow&type=commits